R/utils.R

```diff
@@ -504,9 +504,18 @@ divider_msg <- function(msg = "", width = 60) {
   return(msg)
 }
 
-reg_exec_match <- function(x, pattern) {
-  m <- regexec(pattern,x)
-  regmatches(x,m)
+reg_exec_match <- function(text, pattern, ...) {
+  regmatches(text, regexec(pattern, text, ...))
+}
+
+# TODO: gregexec is now in base R as of 4.1.0
```

Comment:
worth only conditionally creating this if R < 4.1.0?

Reply:
I wondered about that too. I had it all worked out with `gregexec` on 4.1 and didn't even realize it was just super new function.  My implementation does essentially what R's does (I followed the pattern), but gets to what I want more directly with a little less flexibility. So I'd rather rely on R to do the heavy lifting, but in the end the maintenance burden is about the same overall. But would make sense if I needed this in more places.